spec/lib/dyph/min_diff_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe Dyph do
+  describe ".min_diff" do
+    let(:left)  { %w[a b c d e f g] }
+    let(:right) { %w[d e f g a b c] }
+    let(:regular_diff) { Dyph::Differ.two_way_diff(left, right) }
+    let(:subject) { Dyph.min_diff(left, right) }
+
+    its(:length) { is_expected.to be < regular_diff.length }
```

Comment:
Hmm, what if instead of just checking that the length is smaller, we check to ensure it matches `Dyph::Differ.two_way_diff(right, left)`? Optionally also checking that `Dyph::Differ.two_way_diff(right, left) < Dyph::Differ.two_way_diff(right, left)` so if the test fails, we know it's because of min_diff and not that two_way_diff perhaps got better?